apps/framework-cli/src/framework/python/scripts/framework_object_serializer.py

```diff
@@ -0,0 +1,73 @@
+import dataclasses
+from enum import Enum
+import json
+import os
+import re
+import sys
+import typing
+import inspect
+
+# Get the path of the current working directory
+# cwd = os.getcwd()
+
+test_dir = '/Users/timdelisle/Dev/igloo-stack/apps/framework-cli/tests/python'
+
+project_root_dir = test_dir
+
+framework_object_dirs = {
+    'models': os.path.join(project_root_dir, 'models'),
+    'flows': os.path.join(project_root_dir, 'flows'),
+    'aggregations': os.path.join(project_root_dir, 'aggregations')
+}
+
+
+
+sys.path.append(framework_object_dirs['models'])
+import simple
+
+simple_contents = dir(simple)
+
+
+objects = {
+    'enums': {},
+    'dataclasses': {}
+}
+
+def serialize_enum(enum):
+    return {'name': enum.name, 'value': enum.value}
+
+def serialize_type(type_):
+    type_str = str(type_)
+    class_match = re.match(r"<class '(.*)'>", type_str)
+    enum_match = re.match(r"<enum '(.*)'>", type_str)
```

Comment:
`.__name__`?

Reply:
Yeah stupid copilot stuff here. It worked so didn't bother looking into the private APIs